.github/workflows/build-main.yml

```diff
@@ -53,12 +53,19 @@ jobs:
         with:
           commit_message: Add dist artifacts
           file_pattern: dist
-  tag:
+  release-please:
     runs-on: ubuntu-latest
-    needs: [lint-and-test, update-dist]
     steps:
-      - uses: actions/checkout@v2
-      - name: autotag
+      - id: release
+        uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          token: "${{ secrets.BUILD_PATH }}"
+          release-type: node
+          package-name: otel-export-trace
+      - if: ${{ steps.release.outputs.release_created }}
+        uses: actions/checkout@v2
+      - if: ${{ steps.release.outputs.release_created }}
+        name: autotag
         uses: butlerlogic/action-autotag@stable
```

Comment:
You don't need any of the other actions anymore: 

You can also auto-publish on npm on new tags if everything is passing FYI: https://github.com/pragmaticivan/nestjs-otel/blob/main/.github/workflows/release-please.yml#L1-L33

Reply:
Github Actions don't use npm registries. it just uses git refs. so what the other actions are doing are creating tags based on the package.json version and updating previous major and minor semver tags to point to the latest. so that way `v1` points to `v1.1.1`.  i _thought_ release-please will just update package.json version and the change log within the release PR. so this way i don't have to manually update package.json version when pushing a release. i just have to merge the release PR. unless i'm not understanding what the action is doing?

Reply:
yeah, sorry I just meant as an example, you can get the tag as output and do anything with git if you want to reduce dependencies. not a blocker or problem here. Closing that comment.

Reply:
ohh, gotcha. ya that is a good idea

Reply:
ok, removed the tagging and the release workflows. so now my assumption is release-please will create a github release and git tag based on the package.json version that it will updated based on conventional commits. and because i'm using a PAT for the token, it should trigger my version-update.yml workflow which will update the major and minor tag versions to point to the latest.